src/components/MessageBar/MessageBar.css

```diff
@@ -10,6 +10,14 @@
   padding: 7px 12px 8px;
 }
 
+.y-message-bar__nowrap {
+  white-space: nowrap;
+}
+
+.y-message-bar__nowrap svg {
```

Comment:
this seems dangerous, there could be an svg element in a Button with icon, or any other type of action component provided. does MessageBar itself just need an extra pixel of right padding?

Reply:
ah looks like the example has an icon next to text. if this is going to happen everywhere an icon is placed next to text then we should fix it at the Icon component level